modules/node_modules/@ncigdc/components/SurvivalPlotWrapper.js

```diff
@@ -156,7 +156,7 @@ const SurvivalPlotWrapper = ({
             {
               <div className="p-value">
                 <div style={styles.pValue}>
-                  {pValue && `Log-Rank Test P-Value = ${pValue.toExponential(2)}`}
+                  {`Log-Rank Test P-Value = ${pValue.toExponential(2)}`}
```

Comment:
I don't think you can get rid of that check, with 1 curve it will be undefined and you don't want to show anything.